### PR TITLE
Relabel migration helper

### PR DESCRIPTION
**Pull request by ProGM**
Starting from @JustinAiken work on #1166 , I rebased that branch.

Now I'll change that code to fit with our new migration API. ;)

Pings:
@cheerfulstoic
@subvertallchris

### CODE REVIEW

**Comment by JustinAiken** [comment]
Heh, only one failing spec after a few months of code drift, not bad 😀

**Comment by ProGM** [comment]
@JustinAiken The master is failing as well with the same issue, so... 0 failing specs 😜

**Comment by cheerfulstoic** [comment]
failure - failure = win!

**Comment by ProGM** [comment]
@JustinAiken @cheerfulstoic @subvertallchris 
I think I finished :)

I'll be glad if you can take a look to it!

File: lib/neo4j/migrations/helpers/relationships.rb [hunk 1]
```diff
@@ -4,6 +4,8 @@ module Helpers
       module Relationships
         extend ActiveSupport::Concern
 
+        DEFAULT_MAX_PER_BATCH = 1000
```
Comment (JustinAiken) [inline lib/neo4j/migrations/helpers/relationships.rb]: Ah nice idea

File: lib/neo4j/migrations/helpers/relationships.rb [hunk 2]
```diff
@@ -28,7 +32,7 @@ def relabel_relation(old_name, new_name, params = {})
 
         def match_relation(label, params = {})
           from = params[:from] ? "(a:`#{params[:from]}`)" : '(a)'
-          to = params[:to] ? "(a:`#{params[:to]}`)" : '(b)'
+          to = params[:to] ? "(b:`#{params[:to]}`)" : '(b)'
```
Comment (JustinAiken) [inline lib/neo4j/migrations/helpers/relationships.rb]: Derp, that makes way more sense!

**Comment by JustinAiken** [comment]
Thanks for finishing this up @ProGM! It looks better in the new API format and helper'd...

Hopefully the next person that updates from neo4j 3.x to the current has an easier time 😀

**Comment by cheerfulstoic** [comment]
:+1:

**Pull request merged**